pkg/serving/client.go

```diff
@@ -0,0 +1,78 @@
+package serving
+
+import (
+	"fmt"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	client_v1alpha1 "github.com/knative/serving/pkg/client/clientset/versioned/typed/serving/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type NamespacedClient struct {
+	client    client_v1alpha1.ServingV1alpha1Interface
+	namespace string
+}
+
+func NewNamespacedClient(client client_v1alpha1.ServingV1alpha1Interface, namespace string) *NamespacedClient {
```

Comment:
Might want to include single line comments for `/lint` so something like:

```go
// NewNamespacedClient creates a new client bound to namespace
func NewNamespacedClient(client client_v1alpha1.ServingV1alpha1Interface, namespace string) *NamespacedClient {
...
}
```

etc.

Reply:
Good point, I will add those. BTW, I plan to split up this monster PR into multiple PRs (based on each other), and extract eg. the part with this "NamespacedClient" in a separate PR and combine it with #134